packages/ui/src/components/Switch/Switch.vue

```diff
@@ -0,0 +1,96 @@
+<script setup lang="ts">
+interface Switch {
+  id: string
+  label?: string
+  checked?: boolean
+  disabled?: boolean
+}
+
+const props = withDefaults(defineProps<Switch>(), {
+  label: '',
+  checked: true,
+  disabled: false,
+})
+</script>
+
+<template>
+  <div class="s-switch">
+    <input class="s-switch-button" type="checkbox" :id="id" :checked="checked" :disabled="disabled" />
+    <label class="s-switch-label" :for="id">{{ label }}</label>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+$color-switch-btn-label: #2d2926;
```

Comment:
You should use theme-bindings here (recommended via windicss `@apply` directive).
Please look at SJsonInput style block